content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,348 @@
+--omit core/v2.Event,core/v2.APIKey,core/v2.User \
+--format wrapped-json \
+--file backup/config_backup.json
+{{< /code >}}
+
+3. Export the API keys and users, for all namespaces:
+
+   {{< code shell >}}
+sensuctl dump core/v2.APIKey,core/v2.User \
+--all-namespaces \
+--format wrapped-json \
+--file backup/apikeys_users_backup.json
+{{< /code >}}
+
+4. Start a new Sensu backend or cluster and confirm that it is running:
+
+   {{< language-toggle >}}
+
+{{< code shell "Single backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster backend-1.example.com=http://10.0.0.1:2380 \
+--etcd-initial-cluster-token backend-1.example.com \
+--etcd-initial-advertise-peer-urls http://localhost:2380 \
+--etcd-advertise-client-urls http://localhost:2379
+{{< /code >}}
+
+{{< code shell "Clustered backend startup" >}}
+sensu-backend start \
+--etcd-initial-cluster backend-1.example.com=http://10.0.0.1:2380,backend-2.example.com=http://10.0.0.2:2380,backend-3.example.com=http://10.0.0.3:2380 \
+--etcd-initial-cluster-token backend-1.example.com \
+--etcd-initial-advertise-peer-urls http://backend-1.example.com:2380 \
+--etcd-advertise-client-urls http://backend-1.example.com:2379
+
+sensu-backend start \
+--etcd-initial-cluster backend-1.example.com=http://10.0.0.1:2380,backend-2.example.com=http://10.0.0.2:2380,backend-3.example.com=http://10.0.0.3:2380 \
+--etcd-initial-cluster-token backend-2.example.com \
+--etcd-initial-advertise-peer-urls http://backend-2.example.com:2380 \
+--etcd-advertise-client-urls http://backend-2.example.com:2379
+
+sensu-backend start \
+--etcd-initial-cluster sbackend-1.example.com=http://10.0.0.1:2380,backend-2.example.com=http://10.0.0.2:2380,backend-3.example.com=http://10.0.0.3:2380 \
+--etcd-initial-cluster-token backend-3.example.com \
+--etcd-initial-advertise-peer-urls http://backend-3.example.com:2380 \
+--etcd-advertise-client-urls http://backend-3.example.com:2379
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+5. Restore the resources from backup with [`sensuctl create`][1]:
+
+   {{< code shell >}}
+sensuctl create -r -f backup/config_backup.json
+{{< /code >}}
+
+6. Recreate users and API keys, using the `backup/apikeys_users_backup.json` file as a reference.
+
+You should see the restored Sensu configuration in the web UI or sensuctl output.
+
+{{% notice note %}}
+**NOTE**: For details about the sensuctl dump command, read [Back up and recover resources with sensuctl](../../../sensuctl/back-up-recover/).
+{{% /notice %}}
+
+## Disaster recovery for external etcd
+
+{{% notice note %}}
+**NOTE**: This process uses the `etcdctl snapshot save` command to create a backup.
+For details about etcd snapshot and restore capabilities, read [etcd's disaster recovery documentation](https://etcd.io/docs/latest/op-guide/recovery/).
+{{% /notice %}}
+
+If you use external etcd for your Sensu instance, follow these steps to create a backup and restore your Sensu configuration:
+
+1. Take an etcdctl snapshot:
+
```

Comment:
I want to add something in this section to explain what is included in an etcdctl snapshot -- all resources and events? No events? I was not able to get an external etcd instance running so I couldn't test this myself.

Reply:
@hillaryfraley everything that is in etcd, essentially. So....

If using Postgres, only config.

If not using Postgres, allthethings.